README.md

```diff
@@ -31,7 +31,9 @@ soon as possible.)
 * Tests: **Coming soon.** We rely on three test suites: devcrt, tr1, and [libcxx][]. We need to replace our current test
 harness, which extensively uses Microsoft-internal machinery.
 
-* Continuous Integration: **Coming soon.** We need tests first.
+* Continuous Integration: **In progress.** We have set up Azure Pipelines to validate changes of the repository.
+However, that infrastructure still requires manual review before building community submitted pull requests, as we

```

Comment:
```suggestion
However, that infrastructure requires manual review before building community submitted pull requests, as we
```

Reply:
I applied the suggestion but as a separate commit since removal of "still" caused a rewrap.